static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -211,8 +211,16 @@ const buildAppNexusTargeting = once(
         formatAppNexusTargeting(buildAppNexusTargetingObject(pageTargeting))
 );
 
+const getCcpaValue = (ccpaState: boolean | null): string => {
+    if (ccpaState === null) {
+        return 'n/a';
```

Comment:
Not sure n/a is the safest option, in case the '/' character needs to be escaped somewhere in the data pipeline. Maybe just 'na'?

Reply:
The fact that it's in a string should escape it properly but I will remove the dash to be on the safe side.